src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -183,88 +296,472 @@ default ThreadMember getThreadMemberById(String id)
     @CheckReturnValue
     RestAction<ThreadMember> retrieveThreadMemberById(long id);
 
+    /**
+     * Retrieves the {@link ThreadMember ThreadMembers} of this thread.
+     *
+     * This requires the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_MEMBERS} intent to be enabled.
+     *
+     * @return a RestAction that resolves into a List of {@link ThreadMember ThreadMembers} of this thread.
+     */
+    //TODO-v5: docs - documentation depends on implementation cleanup.
     @CheckReturnValue
     RestAction<List<ThreadMember>> retrieveThreadMembers();
 
+    /**
+     * Whether the current account is the owner of this thread.
+     *
+     * @return true if the self account is the owner of this thread, false otherwise.
+     */
     default boolean isOwner()
     {
         return getJDA().getSelfUser().getIdLong() == getOwnerIdLong();
     }
 
+    /**
+     * Gets the ID of the owner of this thread as a long.
+     *
+     * @return the ID of the member who created this thread as a long.
+     */
     long getOwnerIdLong();
 
+    /**
+     * Gets the {@link User} of the owner of this thread as a String.
+     *
+     * @return The {@link User} of the member who created this thread as a String.
+     */
     @Nullable
     default String getOwnerId()
     {
         return Long.toUnsignedString(getOwnerIdLong());
     }
 
+    /**
+     * Gets the {@link User} of the owner of this thread.
+     * <br>
+     * This will be null if the member is not cached, and so it is recommended to retrieve this member from the guild.
```

Comment:
```suggestion
     * This will be null if the member is not cached, and so it is recommended to {@link Guild#retrieveMemberById(long) retrieve this member from the guild} using {@link #getOwnerIdLong()}.
```